packages/hash/api/src/graphql/createApolloServer.ts

```diff
@@ -93,13 +93,14 @@ export const createApolloServer = ({
                 operation: willSendResponseCtx.operationName,
               };
               if (willSendResponseCtx.errors) {
-                const stack = willSendResponseCtx.errors.map(
-                  (err) => err.stack,
-                );
                 willSendResponseCtx.logger.error({
                   ...msg,
                   errors: willSendResponseCtx.errors,
-                  stack,
+                  stack: willSendResponseCtx.errors
+                    .map((err) => err.stack)
+                    .filter(
+                      (stack) => stack && !stack.startsWith("ForbiddenError"),
```

Comment:
It might be helpful to clarify this case in a comment. It's not very clear that `ForbiddenError` relates to Apollo in our case. Happy for you to consider this in a follow-up PR.

Reply:
Have added clarification to what is being filtered here: https://github.com/hashintel/hash/pull/322/commits/c261e725476b558acb79a4d04f575c1162617805